src/view/components/profiler/flamegraph/modes/patchTree.ts

```diff
@@ -46,7 +46,7 @@ export function getStartPosition(tree: Tree, flame: FlameTree, node: DevNode) {
 	//  - otherwise: parent start
 	const idx = parent.children.indexOf(node.id);
 	return idx > 0
-		? flame.get(parent.children[idx - 1])!.end
+		? flame.get(parent.children[idx - 1])?.end
```

Comment:
This is solving the issue but I'm not sure why the prev sibling is not present in the `flame` map

<img width="1172" alt="image" src="https://user-images.githubusercontent.com/11670923/168003599-145961cc-e1ec-47ac-904f-5d4e26e3de11.png">

Reply:
It seems the prev sibling is no longer added to the `flame` map because we're not adding the `Memo(Foo)` node anymore. We only receive the memorized node in `patchTree` here but we never include it in the `flame` map.

https://github.com/preactjs/preact-devtools/blob/696c2df2a20a6a6fddd5feeb7d4dbe8bcffc57f7/src/view/components/profiler/flamegraph/modes/patchTree.ts#L346

Reply:
FYI: Probably unrelated, but looking at the screenshot you're testing against the upcoming Preact 11 for which the devtools bridge isn't complete yet. Does the same error occur with Preact 10?

Reply:
Good catch! @marvinhagemeister same error with Preact 10

Reply:
It seems this change is excluding the memo HoC
https://github.com/preactjs/preact-devtools/commit/e1d1d9b9c247418122add1ab1ef70de8fa6c4841